neoutils/batch_cypher_runner.go

```diff
@@ -81,8 +82,7 @@ func processCypherBatch(bcr *BatchCypherRunner, currentQueries []*neoism.CypherQ
 			}{}
 
 			if jsonErr := json.Unmarshal([]byte(neoErr.Message), &neoErrMsg); jsonErr != nil {
-				log.WithError(jsonErr).Error("ERROR Got error trying to process Neo Error Message")
-				return err
+				return fmt.Errorf("failed to process Neo error message %w", err)
```

Comment:
A ":" after "message" in order to separate the two errors would be useful.

Reply:
It was this way but the linter complains:
"error strings should not end with punctuation or a newline"